client/verta/verta/endpoint/nvidia_gpu.py

```diff
@@ -0,0 +1,85 @@
+from enum import Enum
+
+
+class NvidiaGPU:
+    """
+    Nvidia GPU resources
+
+    The JSON equivalent for this is:
+
+    .. code-block:: json
+
+        {
+            "resources": {"nvidia_gpu": {"number": 1, "model": "V100"}}
+        }
+
+    Parameters
+    ----------
+    number: int > 0
+        Number of GPUs requested to run this endpoint's model.
+    model: :class:`NvidiaGPUModel`, optional
+        Model of GPU requested to run this endpoint's model. Currently available models are
+        specified in the `NvidiaGPUModel` enum.
+
+    Examples
+    --------
+    .. code-block:: python
+
+        from verta.endpoint.resources import Resources, NvidiaGPU, NvidiaGPUModel
```

Comment:
I like this `import`, structure, but it looks like this isn't what we've actually implemented.

We should restructure our files from

```
verta/endpoint/
    nvidia_gpu.py
    resources.py
```

to

```
verta/endpoint/resources/
    __init__.py  # for example see https://github.com/VertaAI/modeldb/blob/main/client/verta/verta/endpoint/__init__.py
    _nvidia_gpu.py
    _resources.py
```